tests/unit/conftest.py

```diff
@@ -58,6 +58,26 @@ def project_with_datasets(project):
     return project
 
 
+@pytest.fixture()
```

Comment:
Would it catch any bugs that wouldn't be caught with Plates in Screens? I assumed it wasn't done for ```project/datasets``` because it was redundant.
Happy to add if it's at-all useful. I guess this would be added in future anyway, when we do Images in Datasets or Wells in Plates?